eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -105,332 +105,396 @@ groups:
       'B->K::alpha^Vt_1np@BFW2010' :
           central: -1.4
           min:     -1.4
           max:     -1.4
+          latex:    '$\alpha_{V_t,1np}^{B \to K,\text{BFW2010}}$'
           comments: ''
       'B->K::alpha^T0_0@BFW2010' :
           central: +0.48
           min:     +0.48
           max:     +0.48
+          latex:    '$\alpha_{T_0,0}^{B \to K,\text{BFW2010}}$'
           comments: ''
       'B->K::alpha^T0_1@BFW2010' :
           central: -1.09
           min:     -1.09
           max:     -1.09
+          latex:    '$\alpha_{T_0,1}^{B \to K,\text{BFW2010}}$'
           comments: ''
       # form factor parameters for B->K inspired by [BSZ2015]
       # new updated results based on a combined LQCD&LCSR fit from the GKvD2018 calculation
       'B->K::alpha^f+_0@BSZ2015' :
           central: +0.32908992854636815
           min:     +0.32908992854636815
           max:     +0.32908992854636815
+          latex:    '$\alpha_{+,0}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^f+_1@BSZ2015' :
           central: -0.8669465867427438
           min:     -0.8669465867427438
           max:     -0.8669465867427438
+          latex:    '$\alpha_{+,1}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^f+_2@BSZ2015' :
           central: +0.006095669673341246
           min:     +0.006095669673341246
           max:     +0.006095669673341246
+          latex:    '$\alpha_{+,2}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^f0_1@BSZ2015' :
           central: +0.19511719957037596
           min:     +0.19511719957037596
           max:     +0.19511719957037596
+          latex:    '$\alpha_{0,1}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^f0_2@BSZ2015' :
           central: -0.4461264576740431
           min:     -0.4461264576740431
           max:     -0.4461264576740431
+          latex:    '$\alpha_{0,2}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^fT_0@BSZ2015' :
           central: +0.2993831492500477
           min:     +0.2993831492500477
           max:     +0.2993831492500477
+          latex:    '$\alpha_{T,0}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^fT_1@BSZ2015' :
           central: -0.7735456824025474
           min:     -0.7735456824025474
           max:     -0.7735456824025474
+          latex:    '$\alpha_{T,1}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       'B->K::alpha^fT_2@BSZ2015' :
           central: +0.009554375102511888
           min:     +0.009554375102511888
           max:     +0.009554375102511888
+          latex:    '$\alpha_{T,2}^{B \to K,\text{BSZ2015}}$'
           comments: ''
       # form factor parameters for B->K according to [KMPW2010], Table 4, p. 31
       'B->K::F^p(0)@KMPW2010' :
           central: +0.34
           min:     +0.32
           max:     +0.39
+          latex:    '$F_+^{B\to K\,,\text{KMPW2010}}(0)$'
```

Comment:
You added a `\,` space here that was not there in the previous parameters, can you make this consistent?